gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -416,15 +423,23 @@ private GlyphRun wrap (BitmapFontData fontData, GlyphRun first, int wrapIndex, i
 		return second;
 	}
 
+	private float getRunOffset (Array<Glyph> glyphs, BitmapFontData fontData) {
+		return glyphs.isEmpty() ? 0 : -glyphs.first().xoffset * fontData.scaleX - fontData.padLeft;
```

Comment:
Shouldn't need `isEmpty`, as we should never have empty runs. See: #6585
The method is OK, since we do it in two places. `getLineOffset` may be a better name, as it's the offset for the first glyph of a line.